src/sentry/logging/audit.py

```diff
@@ -1,74 +0,0 @@
-"""
```

Comment:
This is no longer the direction we're heading. Removed the additional logger and it's tests.